kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/config/S3ConfigSettings.scala

```diff
@@ -138,4 +138,9 @@ object S3ConfigSettings {
   val SOURCE_ORDERING_TYPE:         String = s"$CONNECTOR_PREFIX.ordering.type"
   val SOURCE_ORDERING_TYPE_DOC:     String = "AlphaNumeric (the default)"
   val SOURCE_ORDERING_TYPE_DEFAULT: String = "AlphaNumeric"
+
+  val SOURCE_DELETE_MODE: String = s"$CONNECTOR_PREFIX.source.delete.mode"
+  val SOURCE_DELETE_MODE_DOC: String =
+    "Cleaning index files for GCP Cloud Storage via the compatible S3 APIs requires individual delete requests. Options are `BatchDelete` or `SeparateDelete`. Defaults to `BatchDelete`."
+  val SOURCE_DELETE_MODE_DEFAULT: String = "BatchDelete"
```

Comment:
This property is labeled "SOURCE" but I thought this feature was for the sink?

Reply:
Perhaps it could be a generic property for the connector, not related to SOURCE or SINK?

```
  val DELETE_MODE: String = s"$CONNECTOR_PREFIX.delete.mode"
```